filament/src/driver/opengl/OpenGLDriver.cpp

```diff
@@ -1492,6 +1493,13 @@ bool OpenGLDriver::isFrameTimeSupported() {
     return mPlatform.canCreateFence();
 }
 
+void OpenGLDriver::getMaxViewportDimensions(uint32_t* width, uint32_t* height) {
```

Comment:
mh... I'd prefer if we don't have to add this method.

Reply:
removed.